extension/ezjscore/classes/ezjscpacker.php

```diff
@@ -515,9 +518,15 @@ static function fixImgPaths( $fileContent, $file )
                        $newMatchPath .= implode( '/', $cssPathSlice ) . '/';
                    }
                    $newMatchPath .= str_replace( '../', '', $match );
-                   $fileContent = str_replace( $match, $newMatchPath, $fileContent );
+                   // set match hash
```

Comment:
CS. (sorry for so many nitpicks) The existing comments all start with an upper case letter and I think it would be best to follow this convention within your additions.